snowshu/core/replica/replica_factory.py

```diff
@@ -68,6 +73,7 @@ def _execute(self,
                                  self.config.source_profile.adapter,
                                  self.config.target_profile.adapter,
                                  threads=self.config.threads,
+                                 retry_count=self.retry_count,
```

Comment:
should we add it to cli?

Reply:
Let's add it in to the `analyze` command for consistency between available options.